test/unexpectedAssetGraph.js

```diff
@@ -71,23 +71,25 @@ module.exports = {
             }
         });
 
-        expect.addAssertion('AssetGraph', 'to contain [no] (asset|assets)', function (expect, subject, value, number) {
-            this.errorMode = 'nested';
-            if (typeof value === 'string') {
-                value = {type: value};
-            } else if (typeof value === 'number') {
-                number = value;
-                value = {};
-            }
-            if (this.flags.no) {
-                number = 0;
-            } else if (typeof number === 'undefined') {
-                number = 1;
-            }
-            expect(subject.findAssets(value).length, 'to equal', number);
-        });
+        expect.addAssertion([
+            '<AssetGraph> to contain [no] (asset|assets) <string|object|number?>',
```

Comment:
I don't think you need any arguments in the assertion pattern, the rest of the things should be covered by the next line. I think it should just be <AssetGraph> to contain [no] (asset|assets).